pkg/osutil/interrupt_unix.go

```diff
@@ -69,6 +69,11 @@ func HandleInterrupts() {
 			os.Exit(0)
 		}
 		syscall.Kill(pid, sig.(syscall.Signal))
+
+		// unlock exit lock since the go runtime doesn't reinstall
```

Comment:
> This should always happen since it's closing the etcd server and etcdmain will os.Exit on etcdserver StopNotify.

Right. True. My last concern is that the exit code will be overwritten by etcd, not 128 + sigNum? Not sure if this is OK.